config/production.france.js

```diff
@@ -4,7 +4,7 @@
 import winston from "winston"
 
 
-const apiBaseUrl = process.env.API_URL || `https://fr.openfisca.org/api/v20`,
+const apiBaseUrl = process.env.API_URL || `https://fr.openfisca.org/api/v21`,
```

Comment:
You can set that directly in your [service definition](https://github.com/openfisca/openfisca-ops/blob/3aead030dbaa0963430bcc9cb35c10ad21f48851/fr.openfisca.org/api/v21/openfisca-web-api-fr21.service)

Reply:
👍 Not exactly in this PR perimeter but I agree, this configuration could be extracted from `legislation explorer`